app/Http/Controllers/AuthController.php

```diff
@@ -130,12 +107,20 @@ public function verifyUnauthorizedAction(VerifyRequest $request): JsonResponse
         $user = UserModel::where('id', $verification->user_id)->firstOrFail();
         $user->email_address_verified_at = new DateTime('now', new DateTimeZone('UTC'));
         $user->saveOrFail();
+        $verification->delete();
 
-        $invites = ProjectInvite::where('email_address', $user->email_address)->get();
+        $invites = ProjectInvite::where('email_address', $user->email_address)->orderBy('created_at', 'desc')->get();
+        $is_first = true;
         foreach ($invites as $invite) {
             $invite = ProjectInvite::where('token', $invite->token)
                 ->where('email_address', $user->email_address)
                 ->first();
+            if ($is_first) {
+                $organisation_id = $invite->project->organisation_id;
+                $user->organisation_id = $organisation_id;
```

Comment:
Not a big deal, but it's a little weird to me to stash the organisation ID in an intermediate variable that's not used elsewhere.

Reply:
done